jake/__main__.py

```diff
@@ -19,7 +19,12 @@
 import sys
 import io
 import logging
-from os import _exit, EX_OSERR, path, mkdir
+import platform
+if platform.system() != 'Windows':
```

Comment:
An idea, rather than do this stuff, why not just use `1` where we want to. I think we are all smart enough to know that's exiting as an error? Cuts down on adding more code just to handle Windows.